sycl/test/graph/graph-explicit-saxpy.cpp

```diff
@@ -0,0 +1,51 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+#include <CL/sycl.hpp>
```

Comment:
```suggestion
#include <sycl/sycl.hpp>
```
I believe `CL/sycl.hpp` is deprecated in SYCL 2020 so we should prefer this in new tests I think.